tutorials/super_blink/super_blink.c

```diff
@@ -0,0 +1,36 @@
+#include "libs/timer/api.h"
+
+#include "blink_config.h"
+#include "libs/can/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+#include <avr/interrupt.h>
+
+volatile uint8_t scaled_timer = 0;
```

Comment:
`scaled_timer` doesn't actually have to be volatile! We will go over volatility on Wednesday. Also, you should make it local to the `main` function. As a rule of thumb, variables should have the smallest scope possible.